exp-server.js

```diff
@@ -0,0 +1,63 @@
+const mongoose = require('mongoose')
+require("dotenv").config()
+const express = require('express')
+const user = require('./models/User')
+
+
+const app = express()
+
+mongoose.connect(process.env.MONGO_URL)
+
+app.listen(process.env.PORT , () =>
+console.log(`app running om port: ${process.env.PORT}`))
+
+app.get( '/Users',(req, res) =>{
+   user.find({}).exec((err, data) => {
+    if(err){
+        res.send('error')
+    }else{
+        res.json('data')
+        console.log(data)
+    }
+   })
+})
```

Comment:
Let's use Promises for better readability.

Reply:
how

Reply:
Something like this:

```js
app.get('/users', (req, res) => {
  User.find()
    .then(data => res.json(data))
    .catch(err => res.json(err));
});
```